src/data/pokemon/form_change_table_pointers.h

```diff
@@ -355,6 +355,7 @@ const struct FormChange *const gFormChangeTablePointers[NUM_SPECIES] =
     [SPECIES_ENAMORUS_THERIAN] = sEnamorusFormChangeTable,
 #endif
 #if P_GEN_9_POKEMON == TRUE
+    [SPECIES_PALAFIN_ZERO] = sPalafinZeroFormChangeTable,
```

Comment:
Palafin Hero should also be added here because...

Reply:
added